### ISSUE TITLE
Allow grammars to be obtained as dependency

### ISSUE THREAD

**Issue by gunnarmorling** [antlr/grammars-v4#1233]
Hi, we're using several of these grammars here in the [Debezium](https://github.com/debezium/debezium) project; currently by copying over the grammar definitions into our own project. This makes it a bit cumbersome to keep track of updates/fixes applied to the upstream files here.

So I'd like to suggest to make the grammars available as a dependency, e.g. as an artifact deployed to Maven Central (one can deploy tar.gz files there for instance). That'd greatly help to obtain the grammars in an automated way and simply update them by updating to the latest version of that dependency. Also non-Java users would benefit from that, as they still could download such archive via HTTP.

What are thoughts of folks here on such approach?

**Comment by teverett** [comment]
@gunnarmorling I like the idea, a lot.   This would help me also, and I presume others.

**Comment by teverett** [comment]
I've updated the example project [antlr4example](https://github.com/teverett/antlr4example) to show how to download grammars from github using download-maven-plugin.

**Comment by Marti2203** [comment]
Can this be closed?

**Comment by KvanTTT** [comment]
I don't think so. Grammar as a dependency it's a good idea, but it's unclear how to implement it in the best way.
